spans.log

```diff
@@ -0,0 +1,388 @@
+SpanData {
```

Comment:
We do not need to commit the log file, right?

Reply:
Correct -- will remove that from the PR.